nx.json

```diff
@@ -1,108 +1,134 @@
 {
-    "tasksRunnerOptions": {
-        "default": {
-            "runner": "nx/tasks-runners/default",
-            "options": {
-                "cacheableOperations": [
-                    "build",
-                    "test",
-                    "lint",
-                    "check-types",
-                    "package",
-                    "prepare",
-                    "build:json",
-                    "compile-pb-js",
-                    "compile-pb-as"
-                ],
-                "parallel": 1
-            }
-        }
+  "tasksRunnerOptions": {
```

Comment:
There are a couple of changes in this file? Were they intentional or a messed up re-base?

Reply:
Lerna+NX did some integration stuff that I'm not sure what exactly was done